skyportal/models/group_joins.py

```diff
@@ -328,3 +330,9 @@
 GroupPublicRelease.update = GroupPublicRelease.delete = (
     accessible_by_group_admins & GroupPublicRelease.read
 )
+
+GroupScanReport = join_model("group_scan_reports", Group, ScanReport)
```

Comment:
I don't see any read constraints on the report table that makes use of this, let's either add it here, or simply open an issue so we don't forget